src/app/modules/core/components/map/map.component.ts

```diff
@@ -150,9 +172,12 @@ export class MapComponent extends deepCopy implements OnInit, AfterViewInit {
             this.reportMap.remove();
       }
         this.reportMap = new L.Map('reportMap', this.options);
-        // add point of interest
+		// add point of interest
+	  	// MarkerMaker icon
+		var blackPin = L.divIcon({className: 'wmm-pin wmm-black wmm-icon-circle wmm-icon-white wmm-size-25'});
```

Comment:
The new pin looks great but it seems the point of the pin is not directly on the click point. It almost looks as the though the pin is centered on the click point???